spec/expression.html

```diff
@@ -183,7 +183,7 @@ <h1>Runtime Semantics: TupleSequenceAccumulation</h1>
             1. Let _next_ be ? IteratorStep(_iteratorRecord_).
             1. If _next_ is *false*, return _sequence_.
             1. Let _nextValue_ be ? IteratorValue(_next_).
-            1. Perform ! AddValueToTupleSequenceList(_sequence_, _nextValue_).
+            1. Perform ? AddValueToTupleSequenceList(_sequence_, _nextValue_).

```

Comment:
If an exception is thrown here, then the iterator should be closed. But this could be fixed up in a follow-on patch.